controller_interface/include/controller_interface/controller_interface_base.hpp

```diff
@@ -102,63 +105,80 @@ class ControllerInterfaceBase : public rclcpp_lifecycle::node_interfaces::Lifecy
-   * controller. Each controller can use individual method to determine interface names that in
-   * simples case have the following format: `<joint>/<interface>`. The method is called only in
-   * `inactive` or `active` state, i.e., `on_configure` has to be called first. The configuration is
-   * used to check if controller can be activated and to claim interfaces from hardware. The claimed
-   * interfaces are populated in the \ref ControllerInterfaceBase::command_interfaces_
-   * "command_interfaces_" member.
-   *
-   * \returns configuration of command interfaces.
-   */
+ * @brief Retrieve the configuration for the controller's required command interfaces.
+ *
+ * This method is used by the `controller_manager` to obtain the set of command interfaces 
+ * utilized by the controller. Each controller may use its own method to determine interface 
+ * names, which in the simplest case follow the format: `<joint>/<interface>`. 
+ *
+ * This method is called only when the controller is in the `inactive` or `active` state, 
+ * meaning `on_configure` must be called first. The configuration is used to verify whether 
+ * the controller can be activated and to claim interfaces from the hardware. 
+ *
+ * The claimed interfaces are populated in the 
+ * `ControllerInterfaceBase::command_interfaces_` member.
+ *
+ * @return Configuration of command interfaces.
+ */
+
   virtual InterfaceConfiguration command_interface_configuration() const = 0;
 
-  /// Get configuration for controller's required state interfaces.
   /**
-   * Method used by the controller_manager to get the set of state interface used by the controller.
-   * Each controller can use individual method to determine interface names that in simples case
-   * have the following format: `<joint>/<interface>`.
-   * The method is called only in `inactive` or `active` state, i.e., `on_configure` has to be
-   * called first.
-   * The configuration is used to check if controller can be activated and to claim interfaces from
-   * hardware.
-   * The claimed interfaces are populated in the
-   * \ref ControllerInterfaceBase::state_interfaces_ "state_interfaces_" member.
-   *
-   * \returns configuration of state interfaces.
-   */
+ * @brief Retrieve the configuration for the controller's required state interfaces.
+ *
+ * This method is used by the `controller_manager` to obtain the set of state interfaces 
+ * utilized by the controller. Each controller may use its own method to determine interface 
+ * names, which in the simplest case follow the format: `<joint>/<interface>`. 
+ *
+ * This method is called only when the controller is in the `inactive` or `active` state, 
+ * meaning `on_configure` must be called first. The configuration is used to verify whether 
+ * the controller can be activated and to claim interfaces from the hardware. 
+ *
+ * The claimed interfaces are populated in the 
+ * `ControllerInterfaceBase::state_interfaces_` member.
```

Comment:
Things like `* \ref ControllerInterfaceBase::state_interfaces_ "state_interfaces_" member.` create hyperlink to that class in doxygen. Is that not the case anymore?